doc/netplan.md

```diff
@@ -808,6 +808,24 @@ Example:
 
      **Requires feature: sriov**
 
+``eswitch-mode`` (scalar) – since **0.104**
+
+:    (SR-IOV devices only) Change the operational mode of the embedded switch
+     of a supported SmartNIC PCI device (e.g. Mellanox ConnectX-5). Possible
+     values are ``switchdev`` or ``legacy``, if unspecified the vendor's
+     default configuration is used.
+
+     **Requires feature: eswitch-mode**
+
+``rebind-virtual-functions-delayed`` (bool) – since **0.104**
```

Comment:
Awkward grammar in this key name.  Even if this is matching a string on the mellanox side, would recommend `delay-virtual-functions-rebind` or `delay-rebinding-virtual-functions` instead.